opm/simulators/wells/StandardWellPrimaryVariables.cpp

```diff
@@ -283,11 +256,19 @@ updateNewton(const BVectorWell& dwells,
     this->processFractions();
 
     // updating the total rates Q_t
-    value_[WQTotal] = value_[WQTotal] - dwells[0][WQTotal] * relaxation_factor_rate;
+    value_[WQTotal] = value_[WQTotal] - dwells[0][WQTotal];
```

Comment:
This looks like it could be written as
```C++
value_[WQTotal] -= dwells[0][WQTotal];
```
Do we not have an `operator-=()` for the types in question here?

Reply:
Sure, we can do that.